microcosm_flask/conventions/registry.py

```diff
@@ -0,0 +1,73 @@
+"""
+Support for registering function metadata.
+
+"""
+from microcosm_flask.operations import Operation
+
+
+REQUEST = "__request__"
+RESPONSE = "__response__"
+QS = "__qs__"
+
+
+def iter_operations(graph, match_func):
+    """
+    Iterate through operations in matches.
+
+    """
+    for rule in graph.flask.url_map.iter_rules():
+        try:
+            operation, obj = Operation.parse(rule.endpoint)
+        except (IndexError, ValueError):
+            # operation follows a different convention (e.g. "static")
+            continue
+        else:
+            # match_func gets access to rule to support path version filtering
+            if match_func(operation, obj, rule):
+                func = graph.flask.view_functions[rule.endpoint]
+                yield operation, obj, rule, func
+
+
+def request(schema):
```

Comment:
These decorators save schema information in the request function. This allows the swagger code to use Flask itself as the registry of all metadata.